buzzmobile/plan/car_pose_mux/car_pose_mux.py

```diff
@@ -1,53 +1,66 @@
 #!/usr/bin/env python
+"""car_pose_mux: determines car_pose (auto or manual) to use given car state.
+
+Subscribes:
+    auto_car_pose CarPose calculated from autonomous system
+    manual_car_pose CarPose calculated from controller system
```

Comment:
As a thing we should maybe do in the future, we can namespace the poses, so instead of manual_ and auto_ we have

`car_pose/manual` and `car_pose/auto`.